org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/factory/IInfoFactory.java

```diff
@@ -34,4 +34,6 @@ public InfoPanel create (Lookup lookup, GridField field,
             boolean multiSelection, String whereClause, int AD_InfoWindow_ID);
 	
 	public InfoWindow create (int AD_InfoWindow_ID); 
+	
+	public InfoWindow create (int AD_InfoWindow_ID, String predefinedContextVariables); 
```

Comment:
please use default method to maintain backward compatibility.

Reply:
public default InfoWindow create (int AD_InfoWindow_ID, String predefinedContextVariables) {
       return create (AD_InfoWindow_ID);
}

Reply:
Hi, @hengsin

Thank you very much!

I'm very ashamed, I didn't know How to use "default" at interface class. 

It became very study.

I will try to rewrite and modify special info windows.

Thanks again!
:-D